.github/tests.yml

```diff
@@ -0,0 +1,25 @@
+name: Run Tests
+
+on:
```

Comment:
Tests usually get run on other branches as well because sometimes a contributor/maintainer would want to verify it works before merging, or because other branches are used for different editions of the software. So we should probably keep this to `on: push` instead.